Added small description (#801)
zenml-io/zenml: docs/book/developer-guide/steps-pipelines/inspecting-pipeline-runs.md

run = example_pipeline.get_run(run_name=...)
```

> Besides steps, each run also has a collection of metadata which can be useful:

Reviewer: step accessing hasn't been introduced yet so would not mention that here.

```suggestion
Each run has a collection of useful metadata which you can access:
```
